Dockerfile

```
@@ -94,26 +94,35 @@ RUN /download-frozen-image-v2.sh /build \
 	hello-world:latest@sha256:be0cd392e45be79ffeffa6b05338b98ebb16c87b255f48e297ec7f98e123905c
 # See also ensureFrozenImagesLinux() in "integration-cli/fixtures_linux_daemon_test.go" (which needs to be updated when adding images to this list)
 
-# Just a little hack so we don't have to install these deps twice, once for runc and once for dockerd
-FROM base AS runtime-dev-cross-false
+FROM base AS cross-false
+
+FROM cross-false AS cross-true
```

Might be clearer to just use `base` here

```suggestion
FROM base AS cross-true
```

---

Done.